### ISSUE TITLE
Need clarification regarding implementation of loss function.

### ISSUE THREAD

**Issue by Aritra93** [T-Chittibabu/ATM-OIBSIP#1]
Hi. I don't have any issues to report. I am writing here since I have no means to contact you. I need a deeper clarification as to why you didn't use the simple Pytorch cross-entropy loss. I know you have explained the issue faced in the dataset regarding multiple captions for similar images. But I want a deeper reason for the solution you have provided in your CLIP notebook.
